include/libpmemobj++/detail/common.hpp

```diff
@@ -50,6 +50,44 @@
 #define POBJ_CPP_DEPRECATED
 #endif
 
+#if defined(LIBPMEMOBJ_CPP_VG_ENABLED)
```

Comment:
#if LIBPMEMOBJ_CPP_VG_ENABLED

These checks should  work with LIBPMEMOBJ_CPP_VG_ENABLED=0